build/lib/eksa_releases.sh

```diff
@@ -21,7 +21,7 @@ function build::eksa_releases::load_bundle_manifest() {
   local -r dev_release=${1-false}
   local -r latest=${2-latest}
   local -r echo=${3-true}
-  oldopt=$-
+  oldopt="$(set +o)"
```

Comment:
is this due to a bash version diff or something?  this is fine for now, but as a followup, lets remove the bash call to this altogether and just "curl" in golang directly. cc @abhay-krishna

Reply:
Since we invoke the function with "bash -c", there is an invalid opt "c" in $-. There is an error when you run "set -$oldopt", because you cannot run "set -c"

Reply:
+1. I am okay with this as a temporary solution, but we can change it to a native URL read in Go